app/models/git/track.rb

```diff
@@ -69,6 +69,26 @@ def concepts
       config[:concepts].to_a
     end
 
+    memoize
+    def indent_style
+      config[:online_editor][:indent_style].to_sym
+    end
+
+    memoize
+    def indent_size
+      config[:online_editor][:indent_size]
+    end
```

Comment:
We should probably have a default for this in case its missing? `2`?

Reply:
This should never be missing due to configlet checking its presence. Still want to add a default?

Reply:
Done